.vsts-ci.yml

```diff
@@ -30,5 +30,5 @@ steps:
       configuration: '$(buildConfiguration)'
       versioningScheme: byPrereleaseNumber
       majorVersion: '1'
-      minorVersion: '1'
+      minorVersion: '3'
```

Comment:
bumped to `1.3.0` since `1.2.0` already released
https://www.nuget.org/packages/SignalR.Orleans/1.2.0